infoschema/metric_table_def.go

```diff
@@ -16,6 +16,16 @@ package infoschema
 // MetricTableMap records the metric table definition, export for test.
 // TODO: read from system table.
 var MetricTableMap = map[string]MetricTableDef{
+	"tidb_ops_total": {
```

Comment:
maybe this is not necessary?

Reply:
this used to test while i'm coding. Should I remove this?

Reply:
Surely you should remove it.

Reply:
ok thx for your commet, i will accept it!